src/main/java/onlinebookstore/repository/role/RoleRepository.java

```diff
@@ -0,0 +1,8 @@
+package onlinebookstore.repository.role;
+
+import onlinebookstore.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+
+}

```

Comment:
Where do you use this repository?

Reply:
Requirements: 
Add Role entity with the following fields:id (Long, PK) name (RoleName, not null, unique) (RoleName is enum)
Add RoleRepository
Add the following fields to the User entity: Set<Role> roles